api/server.spec.js

```diff
@@ -0,0 +1,50 @@
+const supertest = require('supertest')
+const server = require('./server');
+const db = require('../database/dbConfig');
+
+beforeEach(async () => {
+  // this function executes and clears out the table before each test
+  await db('users').truncate();
+});
+
+test('welcome route', async () => {
+  const res = await supertest(server).get('/')
+    expect(res.status).toBe(200)
+    expect(res.type).toBe('application/json')
+    expect(res.body.message).toMatch(/Sprint Challenge Authentication/i)
+});
+
+test('add a user to the database', async () => {
```

Comment:
I personally would change what you're saying is happening with this test.  It's only testing what is returned through the register route, but there's no tests actually getting the data from the db